### PR TITLE
Impalaconnection

### PR DESCRIPTION
**Pull request by ksirmons**
### Describe your changes:

Fixes <issue-number>

I worked on impala connection because there are enough differences between it and hive that its own connection was warranted, 

#
### Type of change:

- [ ] Bug fix
- [ ] Improvement
- [x] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] My PR title is `Fixes <issue-number>: <short explanation>`
- [ ] I have commented on my code, particularly in hard-to-understand areas. 
- [ ] For JSON Schema changes: I updated the migration scripts or explained why it is not needed.

### CODE REVIEW

File: ingestion/src/metadata/ingestion/source/database/impala/connection.py [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+
+    options = get_connection_options_dict(connection)
+    if options:
+        params = "&".join(
+            f"{key}={quote_plus(value)}" for (key, value) in options.items() if value
+        )
+        url = f"{url}?{params}"
+    if connection.authOptions:
+        url = f"{url};{connection.authOptions}"
+    print(url)
```
[the remark this answers is not in the document]
Comment (ksirmons) [reply ingestion/src/metadata/ingestion/source/database/impala/connection.py:72]: I'll remove the print statement and push back to my repo

File: ingestion/setup.py [hunk 2]
```diff
@@ -177,6 +181,12 @@ def get_long_description():
         "thrift-sasl~=0.4",
         "impyla~=0.18.0",
     },
+    "impala": {
+        *COMMONS["impala"],
```
[the remark this answers is not in the document]
Comment (ksirmons) [reply ingestion/setup.py:185]: I am not too sure what you are saying here.  I was following the pattern that was setup in Hive, but I didn't fully understand why that pattern was there.  

One thing that may be missing in this setup is the kerberos option.  

`pip install impyla[kerberos]`

Let me know how you want the pip packages to work and I will happily make the changes..

Keith

File: openmetadata-ui/src/main/resources/ui/public/locales/en-US/Database/Impala/fields/authOptions.md
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/public/locales/en-US/Database/Impala/fields/authOptions.md:1]: Hello @ksirmons, Thanks for the contribution, this structure is deprecated and we are following a new structure, you can check it out [here](https://github.com/open-metadata/OpenMetadata/tree/main/openmetadata-ui/src/main/resources/ui/public/locales/en-US/Database).

Everything is in a single file now.

**Review by Sachin-chaurasiya — CHANGES_REQUESTED** [review]
@ksirmons, There is some conflict, please resolve it. Left a few comments, thanks

File: openmetadata-ui/src/main/resources/ui/public/locales/en-US/Database/Impala/fields/authOptions.md
Comment (ksirmons) [reply openmetadata-ui/src/main/resources/ui/public/locales/en-US/Database/Impala/fields/authOptions.md:1]: Updated documentation to a single file

**Review by pmbrull — APPROVED** [review]

**Pull request merged**